### PR TITLE
Toddbell/java server test fix

### PR DESCRIPTION
**Pull request by toddbellMSFT**
While updating the Java SDK for minecraft earth. Our tests were originally expecting a null return where we now return a detailed PlayFab error.

### CODE REVIEW

File: targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs [hunk 1]
```diff
@@ -455,10 +460,8 @@ public class PlayFabApiTest
         errRequest.FunctionName = "throwError";
         errRequest.PlayFabId = playFabId;
         PlayFabResult<PlayFabServerModels.ExecuteCloudScriptResult> errResult = PlayFabServerAPI.ExecuteCloudScript(errRequest);
-        VerifyResult(errResult, true);
-        assertTrue(errResult.Result.FunctionResult == null);
-        assertNotNull(errResult.Result.Error);
-        assertEquals(errResult.Result.Error.Error, "JavascriptException");
+        String errorMessage = CompileErrorsFromResult(errResult);
```
Comment (pgilmorepf) [inline targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:463]: Does errResult.GenerateErrorReport not exist?

File: targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs [hunk 2]
```diff
@@ -434,14 +434,19 @@ public class PlayFabApiTest
     @Test
     public void CloudScriptServer()
     {
+        PlayFabServerModels.LoginWithServerCustomIdRequest customIdReq = new PlayFabServerModels.LoginWithServerCustomIdRequest();
+        customIdReq.CreateAccount = true;
+        String serverId = playFabId;
```
Comment (pgilmorepf) [inline targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:439]: No need for a temporary variable.

File: targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs [hunk 3]
```diff
@@ -434,14 +434,19 @@ public class PlayFabApiTest
...
+        customIdReq.ServerCustomId = serverId;
```
Comment (pgilmorepf) [inline targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:440]: This should use the buildIdentifier the same way the client tests use.
If the server uses a fixed ID, then multiple simultaneous runs of this test will break, all trying to simultaneously access the same player.

File: targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs (hunk 1 above)
Comment (toddbellMSFT) [reply targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:463]: Nope, it looks like similar logic is in the CompileErrorsFromResult function (takes the error message and appends all error details to a single string).

**Pull request merged**